src/components/forms/editSiteForm/index.tsx

```diff
@@ -10,16 +10,29 @@ import styled from 'styled-components';
 
 interface EditSiteFormProps {
   readonly formInstance: FormInstance<EditSiteRequest>;
+  readonly onEdit: (formLat: number, formLng: number) => void;
 }
 
 const EditForm = styled(Form)`
   width: 100%;
 `;
 
-const EditSiteForm: React.FC<EditSiteFormProps> = ({ formInstance }) => {
+const EditSiteForm: React.FC<EditSiteFormProps> = ({
+  formInstance,
+  onEdit,
+}) => {
   return (
     <>
-      <EditForm name="basic" form={formInstance}>
+      <EditForm
+        name="basic"
+        form={formInstance}
+        onValuesChange={(changedValues, allValues) => {
```

Comment:
🔧 Could we move this function outside of the return statement for readability purposes? Sort of like:

```
const EditSiteForm = ({ formInstance, onEdit }) => {
  const onValuesChange = (changedValues, allValues) => {
    ...
  }

  return (
    <>
      <EditSiteForm ... onValuesChange={onValuesChange}>
    ...
    </>
  )
}
```